ukis_csmask/mask.py

```diff
@@ -52,6 +54,13 @@ def __init__(
         if img.dtype != np.float32:
             raise TypeError("img must be in top of atmosphere reflectance with dtype float32")
 
+        if img.shape[0] != 256 or img.shape[1] != 256:
+            warnings.warn(
+                message=f"The model has been trained and is optimized for 256x256 image tiles, you are using "
```

Comment:
Perhaps slightly rephrase the message `message=f"Your input image is smaller than the internal tiling size of 256x256 pixels. This may result in suboptimal performance. Consider using a larger image size."`